app/views/books/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1>Books</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Year of publication</th>
+      <th>Wikipedia url</th>
+      <th>Goodreads url</th>
+      <th colspan="2">Author</th>
+      <th colspan="3">Actions</th>
+    </tr>
+    <% @books.each do |book| %>
+    <tr>
+      <td><%= book.title %></td>
+      <td><%= book.year_of_publication %></td>
+      <td><%= book.wikipedia_url %></td>
+      <td><%= book.goodreads_url %></td>
+      <td><%= book.author.first_name %></td>
+      <td><%= book.author.last_name %></td>
```

Comment:
let's add method full_name for Author model witch would look like this:

```
def full_name
  "#{first_name} #{last_name}"
end
```

and use it here, so that to have 1 <td>.

please, add a test for method full_name to verify that it returns correct value